src/main/resources/pgsql/setup.sql

```diff
@@ -0,0 +1,12 @@
+create schema debezium;
+
+create table debezium.customer (id int primary key, fullname varchar(255), email varchar(255));
+create table debezium.car_model (id int primary key, model varchar(255), brand varchar(255));
+
+create user gingersnap_user with password 'password';
+alter user gingersnap_user REPLICATION LOGIN;
+
+grant create on database debeziumdb to gingersnap_user;
+grant usage, create on schema debezium to gingersnap_user;
+
+alter table debezium.customer owner to gingersnap_user;
```

Comment:
Does our user have to own the DB table?

Reply:
Yes :( 
The 4th step https://debezium.io/documentation/reference/2.0/connectors/postgresql.html#postgresql-replication-user-privileges